### PR TITLE
add new spec for vma_ra_enabled_s390x

### PR DESCRIPTION
**Pull request by xhe123**
New spec for vma_ra_enabled_s390x

Signed-off-by: Xiaonan He <xhe@redhat.com>

### CODE REVIEW

File: uploader.json [hunk 1]
```diff
@@ -3078,6 +3078,11 @@
             ],
             "symbolic_name": "virtlogd_conf"
         },
+        {
+            "file": "/sys/kernel/mm/swap/vma_ra_enabled",
+            "pattern": [],
+            "symbolic_name": "vma_ra_enabled_s390x"
```
Comment (bfahr) [inline uploader.json]: @xhe123 the symbolic name needs to match the spec name in insights core, `vma_ra_enabled`.

File: uploader.v2.json [hunk 2]
```diff
@@ -3013,6 +3013,11 @@
             ],
             "symbolic_name": "virtlogd_conf"
         },
+        {
+            "file": "/sys/kernel/mm/swap/vma_ra_enabled",
+            "pattern": [],
+            "symbolic_name": "vma_ra_enabled_s390x"
```
Comment (bfahr) [inline uploader.v2.json]: See comment above.

**Comment by xhe123** [comment]
@bfahr thanks for your advice. Will you please have a look again?

I resubmitted this PR against the uploader_json branch, I changed the symbolic_name to the same as SPEC name now.

**Comment by bfahr** [comment]
@xhe123 the symbolic_name looks correct, but you need to resubmit this PR against the `uploader_json` branch instead of the `master` branch.

**Comment by xhe123** [comment]
@bfahr I did rebase on upload_json branch and resubmitted it, will please review? Thanks!

**Comment by bfahr** [comment]
@xhe123 the PR is still submitted against the `master` branch.  It cannot be merged until you submit the PR against the `uploader_json` branch.  At the top of this page you should see `xhe123 wants to merge 1 commit into RedHatInsights:uploader_json from xhe123:vma_ra_enabled_s390x`.

**Comment by xhe123** [comment]
@I changed the target branch on upstream to upload_json, it should work.

**Pull request merged**